docs/data/joy/pages.ts

```diff
@@ -142,6 +142,10 @@ const pages: readonly MuiPage[] = [
             pathname: '/joy-ui/customization/overriding-component-structure',
             title: 'Overriding the component structure',
           },
+          {
+            pathname: '/joy-ui/customization/right-to-left',
```

Comment:
I'm not sure this is about customization, in the sense of implementing a custom design system. It feels more like a general guide.

Reply:
I feel like it does fit into the Customization category because it requires more effort/instructions to make it work — as opposed to just "customizing" the component with a built-in prop — and it's different from how the library is by default. @samuelsycamore — curious if you have any thoughts!

Reply:
I think it makes sense within the Guides subsection of the Customization section. Given the current info architecture we're working with here, I'm not sure where else it could go. It _does_ involve customizing the theme, so I think it fits in Customization, but it's kind of tangential to other theme-related topics so I don't think I'd put it in that subsection. As a quick benchmark, Chakra UI places their "RTL Support" page within their "Styled System" section, which is more or less equivalent to our Customization section in terms of the content you'll find there.